lib/sql/where.js

```diff
@@ -16,9 +16,6 @@ class Operand {
 
 class And extends Operand {
   toSQL (values) {
-    if (this.children.length === 0) {
-      return '1'
-    }
```

Comment:
🔴 This might still be necessary:

```javascript
Book.objects.filter({})
```

would produce a non-empty root `where` clause. (We should def add a test for this!)

Reply:
this is covered because `getWhereClause()` (the one place that calls `toSQL()`) already tests `children.length` and returns an empty string when it's 0 here: https://github.com/chrisdickinson/ormnomnom/blob/test_refactor/lib/sql/builder.js#L253-L255

although now that i think about it, i bet something like `filter([{}])` would mess it up